scripts/add_electricity.py

```diff
@@ -179,7 +169,7 @@ def costs_for_storage(store, link1, link2=None, max_hours=1.):
         costs_for_storage(costs.loc["battery storage"], costs.loc["battery inverter"],
                           max_hours=max_hours['battery'])
     costs.loc["H2"] = \
-        costs_for_storage(costs.loc["hydrogen storage"], costs.loc["fuel cell"],
+        costs_for_storage(costs.loc["hydrogen storage underground"], costs.loc["fuel cell"],
```

Comment:
`hydrogen storage underground` is only in `resource/costs.csv`
which is different to
`hydrogen underground storage` that is only in `data/costs.csv`

I would recommend just renaming the name in data/costs.csv to hydrogen storage underground

Reply:
A few lines above you need also something like this in case your tutorial package has `costs.csv`:

  # because technology data & pypsa earth costs.csv use different names
  # TODO: rename the technologies in hosted tutorial data to match technology data
  costs = costs.rename(
      {
          "hydrogen storage": "hydrogen storage tank",
          "hydrogen storage tank": "hydrogen storage tank",
      },
  )